### DISCUSSION TITLE
Question about FilamentPHP forms and table elements.

### DISCUSSION THREAD

**Discussion by emiliosh** [filamentphp/filament#8466] [Help] [answered]
I have a Select in my form which I use for hiding or showing different elements but It is not related to any field in my model.

When I try to save, obviously I receive a  **SQLSTATE\[42703\]: Undefined column: 7 ERROR,** because the name I used in Select::make was something random in order to control the flow of the form.

What should I do?  How can I solve this?  I have checked the documentation and the source code but wasn't able to find out how to do it.

Thanks.

PS:  I have some similiar problem with "custom tailored column" in Table because I have to use make with a real field instead something random.

**Comment by leandrocfe** [comment, marked as answer]
[dehydrated](https://filamentphp.com/docs/3.x/forms/advanced#preventing-a-field-from-being-dehydrated) ?

**Reply by emiliosh** [reply]
Thx for your fast response, yes, that was the answer.

**Comment by emiliosh** [comment]
Thank you.
